Bottom: 53f00af95df12c3aa07ab0835c666a9645b86b7f
Top:    8b6bb37c12c4672472e625ad648944ae8c0ebf16
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-29 16:23:51 -0700

Refresh of addrinfo-af-ib

---

diff --git a/src/addrinfo.c b/src/addrinfo.c
index 01c04d1..f606470 100755
--- a/src/addrinfo.c
+++ b/src/addrinfo.c
@@ -122,6 +122,7 @@ static int ucma_convert_in6(struct sockaddr_ib **dst, socklen_t *dst_len,
 		return ERR(ENOMEM);
 
 	(*dst)->sib_family = AF_IB;
+	(*dst)->sib_pkey = 0xFFFF;
 	(*dst)->sib_flowinfo = src->sin6_flowinfo;
 	ib_addr_set(&(*dst)->sib_addr, src->sin6_addr.s6_addr32[0],
 		    src->sin6_addr.s6_addr32[1], src->sin6_addr.s6_addr32[2],
